bundler.d/example.rb

```diff
@@ -6,6 +6,8 @@
 # or `ensure_gem` to make sure a gem is there w/o worrying about if it is an
 # override or not
 
-ensure_gem "sentry-ruby"
-ensure_gem "sentry-rails"
+gem "sentry-ruby"
+gem "sentry-rails"
 ensure_gem "cancancan", "~> 3.0" # cancancan is bundling to v1.17.0 but we need at least 3.0
+
+ensure_gem "willow_sword", github: "notch8/willow_sword", ref: "bd43991e8e1b0c660fc29f94e7fec38e8ca03e1c"
```

Comment:
You can play around with override gem or ensure gem I was just trying to get something to work.